src/gui/newwizard/syncmode.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+namespace OCC::Wizard {
+
+enum class SyncMode {
+    Invalid = -1,
+    SyncEverything = 0,
+    SelectiveSync,
+    UseVfs,
+};
+
```

Comment:
Q_ENUM_NS() so we have the qDebug operator if needed.
You'll need to include syncmode.moc in some other .cpp.

Or... you move it to one of the QObjects.

If invalid is needed please start counting a 0, no need for the magic here.